content/chat/rooms/typing.textile

```diff
@@ -199,13 +205,49 @@ try await room.typing.stop()
 room.typing.stop()
 ```
 
+h3(#frequency). Typing Event Frequency
+
+The Typing feature includes a configurable timer that controls how often typing events are sent to the server. This timer is reset each time a new typing event is sent, it works as follows:
+- On the **first call** to @start()@, the timer is set and an event is sent to the server.
+- **Subsequent calls** before the timer expires result in a no-op.
+- After the timer expires, a new typing event is sent and the timer is reset.
+- If @stop()@ is called, the timer is reset and a @typing.stopped@ event is sent to the server.
+
+You can configure the length of this timer using the @heartbeatThrottleMs@ parameter in @RoomOptions@ (default: **10,000ms**).
+It is recommended that you call @start()@ with every keypress, and the SDK will handle when and if to send a typing indicator to the server.
+
+h3(#emulating-heartbeats). Emulating User Behavior
+
+If needed, you can emulate user behavior (e.g., in chatbots) by setting a timeout to call @start()@ at intervals equal to the @heartbeatThrottleMs@ plus a small delay, e.g. 200ms. This will ensure the typing indicator remains active.
```

Comment:
```suggestion
You can emulate user behavior (e.g., in chatbots) by setting a timeout to call @start()@ at intervals equal to the @heartbeatThrottleMs@ plus a small delay, e.g. 200ms. This will ensure the typing indicator remains active.
```